Topics/Tech_Stacks/TypeScript.md

```diff
@@ -67,13 +80,18 @@ To use TypeScript in your Vue 3 project using `create-vue`, which is the officia
 
 3. Follow the command line prompts to select the preferences for your project. Select `yes` for the prompt `Add TypeScript?`, and manage other settings to your preference.
 
+<img width="450" src="https://raw.githubusercontent.com/vuejs/create-vue/main/media/screenshot-cli.png"> [image source](https://github.com/vuejs/create-vue)
+
 4. Done! The Vue project is succesfully created.
 
 It is worth noting that this method is the preferred way to create Vue projects, instead of using Vue CLI, since create-vue is based on Vite. For more information, visit the create-vue Github repository [here](github.com/vuejs/create-vue).
+
+However, if you wish to set up your Vue project with Vue CLI, you can do so by following the instructions in [this video](https://youtu.be/JfI5PISLr9w?si=HEKZJlcRhWjb0F_0).
+
```

Comment:
I think it would be interesting to provide some examples of code where JS and TypeScript are compared: for  example, you could have the same simple program written in both languages and use that to show how TypeScript is easier to understand.